crates/matrix-sdk/src/client.rs

```diff
@@ -951,7 +952,6 @@ impl Client {
         /// The number of times the SSO server will try to bind to a random port
         const SSO_SERVER_BIND_TRIES: u8 = 10;
 
-        info!("Logging in to {}", self.homeserver().await);
```

Comment:
Why not move the await out of the `info!()` call?

Reply:
added it back :)

Reply:
Yeah, this is https://github.com/tokio-rs/tracing/issues/1487 and should be fixed by simply moving out that `self.homeserver().await`.